### ISSUE TITLE
command incompatibility with common plugins

### ISSUE THREAD

**Issue by fallerOfFalls** [RichardsProjects/AutismChat3#31]
Since autismchat was initially designed for a very vanilla server, I haven't been too careful about command conflicts.

I'm currently aware of one conflict:
/gc is an essentials command that displays server usage information.

We probably don't need to fix this unless someone complains.

**Comment by RichardB122** [comment]
How about we make /gc an option in the config (that by default would be false) and the default command would /globalchat. If a server doesn't use essentials they could then enable /gc.

**Comment by fallerOfFalls** [comment]
Yes, that's a good idea. But /gc should be _enabled_ by default since the plugin is primarily intended for small, relatively vanilla servers.

**Comment by fallerOfFalls** [comment]
If we are changing "global chat" to "announce" in #58 then we should change /gc to something else. Or perhaps remove it entirely? I will need to think on this.
